# широта, [градусы], [-90, 90]
float64 lat

# долгота, [градусы], [-180, 180]
float64 lon